### PR TITLE
[fix](cloud) ignore black list in cloud mode

### PR DESCRIPTION
**Pull request by yujun777**
Cloud will auto change replica's backend when it found backend is dead. No need to use black list in cloud mode.

## Proposed changes

Issue Number: close #xxx

### CODE REVIEW

**Comment by yujun777** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/qe/SimpleScheduler.java [hunk 1]
```diff
@@ -171,6 +171,10 @@ private static String getBackendErrorMsg(List<Long> backendIds, ImmutableMap<Lon
     }
 
     public static void addToBlacklist(Long backendID, String reason) {
+        if (Config.isCloudMode()) {
+            return;
```
Comment (gavinchou) [inline fe/fe-core/src/main/java/org/apache/doris/qe/SimpleScheduler.java:175]: Log here if this is called in cloud mode

**Review by gavinchou — APPROVED** [review]

**Comment by yujun777** [comment]
run buildall

**Review by deardeng — APPROVED** [review]
LGTM

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**